plugins/catalog.nix

```diff
@@ -13,6 +13,7 @@
   # mountpoint in catelog attr
   # catalog.<system>.<stability>[.<path>].<catalog entries>
   path ? [],
+  pathPrefix ? [],
```

Comment:
Calling this `pathPrefix` makes me think it will result in `catalog[.<pathPrefix>].<system>.<stability>[.<path>].<catalog entries>` Is there a better name that would convey more of what this one specifically is for? `allowedPathPrefix`? `catalogPaths`? `allowedCatalogPaths`?

Reply:
- knownPathPrefix
- includePaths

Those sound good. I don't have a strong preference or opinion.